addons/mail/static/src/models/composer_view/composer_view.js

```diff
@@ -41,6 +45,20 @@ registerModel({
         closeSuggestions() {
             this.update({ suggestionDelimiterPosition: clear() });
         },
+        /**
+         * Returns whether the given html node is inside this composer view,
+         * including whether its inside the emoji popover when active.
+         *
+         * @param {web.HtmlNode} node
```

Comment:
it's just an html node. I just avoided uppercase of acronym "HTTP" (for readability concerns), and the `web.` prefix is just to clarify it's a typing in the Web API, to avoid confusion with types that are models.

Reply:
But the actual name of this interface is just `Node` so having Html in the name is giving the wrong idea that is more complex than it really is. And the web prefix is confusing with our web module.

Also is it receiving any generic `Node`? It is probably always `Element`? Note that `HTMLElement` also exists but it is probably too specific here (if we want to "support" svg elements too for example).

All of this to say we are probably better to stick with just `{Element}` like we usually do. Ok there is confusion possible with a model name, except we don't have a model named that way, and we probably should never name a model that way as this name does not represent any clear concept by itself. I feel trying to avoid confusion with model name is adding confusion with everything else here.

Reply:
> [...] Ok there is confusion possible with a model name, except we don't have a model named that way [...]

I have intention to introduce an `Element` model that is different from Web API's `Element`. I made the difference with our models being named `TheModel` and web stuffs named `web.TheModel`.

For me, implicitly all our models are `odoo.TheModel`. It's just that I omit it because it's too noisy and it's our code so make sense that our things have priority over the rest.

Anyway, for this comment, I set `{Element}`. Just wanted to let you know the thought process being this naming.